scripts/exregional_aqm_lbcs.sh

```diff
@@ -220,6 +282,7 @@ The executable (exec_fp) for GEFS LBCs does not exist:
   exec_fp = \"${exec_fp}\"
 Please ensure that you've built this executable."
   fi
+  #
```

Comment:
@JianpingHuang-NOAA, please remove this line.

Reply:
Done